src/MeMoModel.py

```diff
@@ -61,24 +61,33 @@ def fromSBML(cls, model: sbml.Model) -> MeMoModel:
         _id = model.getId()
         return MeMoModel(metabolites=metabolites, _id=_id)
 
-    def annotate(self) -> None:
+    def annotate(self) -> AnnotationResult:
         """Goes through the different bulk annotation methods and tries to annotate InChI strings to the metabolites
         in the model"""
+        print(self._id)
+
```

Comment:
Currently, we do not use the origin_database here, this should be changed before merge